release/_bom.sh

```diff
@@ -76,6 +76,37 @@ function generate_api_jars {
 	do
 		_manage_bom_jar "${module_jar}"
 	done
+
+	for file in $(ls api-jar/META-INF --almost-all | grep --extended-regexp --invert-match '^(alloy-util.tld|alloy.tld|c.tld|liferay.tld)$')
+	do
+		if [[ "$file" == *.tld ]]
+		then
+			rm "api-jar/META-INF/${file}"
+		fi
+	done
+
+	local tld_files=($(find "${_PROJECTS_DIR}" -name "liferay-*.tld" -o -name "ratings.tld" -type f | awk -F/ '{print $NF, $0}' | sort -k1,1 -u | awk '{print $2}'))
+
+	for file in "${tld_files[@]}"
```

Comment:
@lucasmiranda0, do we really need to declare the variable `tld_files`? Can't we use the content of it directly here?

For instance:
```
	for file in $(find "${_PROJECTS_DIR}" \
		-name "liferay-application-list.tld" -o \
		-name "liferay-data-engine.tld" -o \
		-name "liferay-ddm.tld" -o \
		-name "liferay-export-import-changeset.tld" -o \
		-name "liferay-form.tld" -o \
		-name "liferay-staging.tld" -o \
		-name "liferay-template.tld" -o \
		-name "react.tld" -o \
		-name "soy.tld" -type f | \
		awk -F/ '{print $NF, $0}' | \
		sort -k1,1 -u | \
		awk '{print $2}')
	do
		cp "$file" api-jar/META-INF/resources
	done
```